server/vulnerabilities/msrc/analyzer.go

```diff
@@ -159,8 +163,16 @@ func patched(
 			continue
 		}
 
-		// Check if the kernel build already contains the fix
-		if utils.Rpmvercmp(os.KernelVersion, fix.FixedBuild) >= 0 {
+		if fix.FixedBuild == "" {
+			continue
```

Comment:
When is this empty?

Should we `continue` here? (What if the host has a patch that fixes the vuln that is not yet part of a fixed build, can this happen?)

Reply:
This looks to be a bug in the MSRC feed where the `FixBuild` key is missing , and the latest occurrence I can find seems to be around 04-2021.  We should continue here in order to prevent false positives.  I added  some comments around that.